WebView/app/src/main/assets/app.css

```diff
@@ -13,15 +13,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-body {
-  background-color: light-blue;
+
+/* Dark Theme */
+@media (prefers-color-scheme: dark) {
+  body {
+     background: #121212;
```

Comment:
nit: how does this look in the app? It might be nice to make this just a little lighter (darkish gray) so it's obvious we're using a custom theme instead of the automatic theme.

Reply:
Great point, I shifted it to #383838 to make that more clear, here is a scale for reference. 
<img width="305" alt="Screen Shot 2020-07-20 at 9 56 32 AM" src="https://user-images.githubusercontent.com/50959941/87945843-4cfc3880-ca6f-11ea-897a-8d7ee257d47b.png">